netplugin/netd.go

```diff
@@ -33,6 +35,7 @@ import (
 	"github.com/samalba/dockerclient"
 
 	log "github.com/Sirupsen/logrus"
+	"github.com/Sirupsen/logrus/hooks/syslog"
```

Comment:
This should already be included in the logrus clone, is it necessary to import it explicitly?

> On May 26, 2015, at 9:44 AM, Madhav Puri <notifications@github.com> wrote:
> 
> In netplugin/netd.go <https://github.com/contiv/netplugin/pull/79#discussion_r31053974>:
> 
> > @@ -33,6 +35,7 @@ import (
> >  	"github.com/samalba/dockerclient"
> >  
> >  	log "github.com/Sirupsen/logrus"
> > +	"github.com/Sirupsen/logrus/hooks/syslog"
> need to update godep for this.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/contiv/netplugin/pull/79/files#r31053974>.
>